src/lib/snack-bar/snack-bar.ts

```diff
@@ -145,6 +148,18 @@ export class MatSnackBar {
     // We can't pass this via the injector, because the injector is created earlier.
     snackBarRef.instance = contentRef.instance;
 
+    // Subscribe to the breakpoint observer and attacg the mat-snack-bar-handset class as
+    // appropriate. This class is applied to the overlay element because the overlay must expand to
+    // fill the width of the screen for full width snackbars.
+    const isHandset = this._breakpointObserver.observe(Breakpoints.Handset).subscribe(state => {
+      overlayRef.overlayElement.classList.toggle('mat-snack-bar-handset', state.matches);
```

Comment:
The second param for `classList.toggle` doesn't work on IE11. This should use `add`/`remove` instead.